src/useAxios.ts

```diff
@@ -20,8 +20,14 @@ function useAxios<Data>(param1: string | Config, param2: Config = {}) {
 
   const [getData, dataStates] = invokeUseBaseAxios();
 
+  const justMounted = useRef(true);
+
   useEffect(() => {
-    getData();
+    if (typeof config.ssrData === 'undefined' || !justMounted.current) {
```

Comment:
Thoughts on `config.ssrData === undefined` vs. `typeof config.ssrData === 'undefined'`?

Reply:
Could you elaborate on the need for `justMounted` ref?

Reply:
I followed this pattern: https://github.com/therealparmesh/use-axios-client/blob/v2.0.0/webpack.config.js#L23

Reply:
We need the `justMounted` ref to skip the initial call to useEffect (when the component mounts) if SSR data is passed in. [I copied this logic from the `useEffectAfterMount` hook that Kent C. Dodds wrote here.](https://kentcdodds.com/blog/compound-components-with-react-hooks)